ext/openssl/tests/ecc.phpt

```diff
@@ -1,7 +1,7 @@
 --TEST--
 openssl_*() with OPENSSL_KEYTYPE_EC
 --SKIPIF--
-<?php if (!extension_loaded("openssl") && !defined("OPENSSL_KEYTYPE_EC")) print "skip"; ?>
+<?php if (!extension_loaded("openssl") || !defined("OPENSSL_KEYTYPE_EC")) print "skip"; ?>
```

Comment:
Thanks for fixing this. Was introduced 2 years ago. https://github.com/php/php-src/commit/9688138d38034c9558af45f1ebcd77c3ff88c420#diff-a2ffa4ffa39980a4fe2fb9a7cb455e6eR4 ;-)